src/main/java/nextstep/subway/line/LineConverter.java

```diff
@@ -9,7 +10,8 @@
 @Component
 public class LineConverter {
     public LineResponse convert(Line line) {
-        List<StationResponse> stationResponses = line.getSections().getStations()
+        List<Station> stations = line.getSections().getStations();
```

Comment:
```suggestion
        List<Station> stations = line.getAllStations()
```

이런식으로 전철역에대한 조회 기능을 노선에게 전가할 수 있을 것 같아요 ㅎㅎ